terraform/s3.tf

```diff
@@ -81,10 +81,10 @@ resource "aws_s3_bucket_ownership_controls" "pcc_master_data_ownership" {
 resource "aws_s3_bucket_public_access_block" "pcc_master_data_pb" {
   bucket = aws_s3_bucket.pcc-master-data.id
 
-  block_public_acls       = false
-  block_public_policy     = false
-  ignore_public_acls      = false
-  restrict_public_buckets = false
+  block_public_acls       = true
```

Comment:
What are these changes for?

Reply:
We don't need public access on the master data bucket. Better to have more restrictions than fewer?